Marker[] markers  # Array of detected markers